images/regen-sandbox/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.18-alpine AS build
```

Comment:
one preference i have is to make use of the bullseye image. obviously alpine results in smaller images, but it doesn't always result in faster build processes depending on what needs to be installed and it definitely doesn't result in drastically better performance at runtime. i also find it easier to find packages with apt-get compared to apk and generally have found it a bit easier to work with. either way, we'll definitely want to update and upgrade all installed packages after this first step: `RUN apk -U upgrade` or `RUN apt-get update && apt-get upgrade -y`